src/ms.tex

```diff
@@ -266,6 +266,46 @@ \subsection{Affiliated packages}
 Include a big table of all affiliated packages and references (as in v2.0
 paper).
 
+All affiliated packages are listed on the astropy 
+website\footnote{\url{https://www.astropy.org/affiliated}}. Since 
+\cite{astropy:2018}, there has been an explosion of gravitational-wave affiliated 
+packages including: \texttt{PyCBC} for exploring GW sources, lenstronomy for 
+modeling strong gravitational lenses, \texttt{ligo.skymap} for visualizing GW 
+probability maps, and \texttt{EinsteinPy} for general relativity and gravitational astronomy. 
+There have been several packages related to HEALPix: \texttt{astropy-healpix}
+for a BSD licensed HEALPix implementation, and \texttt{mocpy} for Multi-Order 
+Coverage maps. \texttt{astroalign} as been introduced for astrometric registration, 
+and \texttt{python-cpl} has been added for ESO pipelines and VLT data products. 
+For ground-based astronomy, \texttt{baseband} has added IO capabilities for 
+VLBI, and \texttt{SpectraPy} brings slit spectroscopy to the astropy ecosystem. 
+We have \texttt{agnpy} for AGN astrophysics, and \texttt{statmorph} for fitting galactic 
```

Comment:
Maybe it would be worth to mention also naima? 
Thanks to naima and agnpy one can model the non-thermal radiative processes of relativistic particles in galactic and extragalactic sources. Also, to be specific, I think "AGN astrophysics" is a bit too general for AGNpy, as it is mostly meant for processes in the jets of AGN - I perfectly understand the text limitations, just wanted to point it out.